src/Raven.Client/Documents/DocumentStore.cs

```diff
@@ -428,12 +428,17 @@ public override OperationExecutor Operations
         }
 
         public override BulkInsertOperation BulkInsert(string database = null, CancellationToken token = default)
+        {
+            return BulkInsert(null, database, token);
+        }
+
+        public override BulkInsertOperation BulkInsert(BulkInsertOptions bulkInsertOptions, string database = null, CancellationToken token = default)
```

Comment:
bulkInsertOptions => options

Change the order of parameters:
```
string database = null, BulkInsertOptions options, CancellationToken token = default
```

Reply:
Cannot have this since an optional parameter must appear after all the required ones.
If I change to:
`string database = null, BulkInsertOptions options = null, CancellationToken token = default`
then we have to ambiguous overloads:
`string database = null, CancellationToken token = default`
`string database = null, BulkInsertOptions options = null, CancellationToken token = default`

Reply:
`string database, BulkInsertOptions options, CancellationToken token = default`

Reply:
done